decred/tests/unit/dcr/test_dcrdata_unit.py

```diff
@@ -128,31 +128,13 @@ def test_checkoutput():
         dcrdata.checkOutput(tx, 0)
 
 
-class MockWebSocketClient:
-    def __init__(self, url, on_message, on_close, on_error):
-        self.on_message = on_message
-        self.on_close = on_close
-        self.on_error = on_error
-        self.sent = []
-        self.received = []
-
-    def send(self, msg):
-        self.sent.append(msg)
-
-    def close(self):
-        self.on_close(self)
-
-    def receive(self, msg):
-        self.received.append(msg)
-
-
 @pytest.fixture
-def tweakedDcrdataClient(monkeypatch):
-    monkeypatch.setattr(ws, "Client", MockWebSocketClient)
+def tweakedDcrdataClient(monkeypatch, MockWebSocketClient_class):
+    monkeypatch.setattr(ws, "Client", MockWebSocketClient_class)
 
     def make():
         ddc = DcrdataClient(BASE_URL)
-        ddc.emitter = ddc.psClient().receive
+        ddc.emitter = ddc.psClient().emit
```

Comment:
Name changed for clarity. This is actually the emitting part, not the receiving one.